### PR TITLE
[MIG][13.0] Migrate product_manufacturer

### PR DESCRIPTION
**Pull request by guewen**

### CODE REVIEW

File: product_manufacturer/views/product_manufacturer_view.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+    <record id="product_template_manufacturer_form_view" model="ir.ui.view">
+        <field name="name">product.template.form - product_manufacturer</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_form_view"/>
+        <field name="arch" type="xml">
+            <page name="general_information" position="inside">
+                <group name="manufacturer" string="Manufacturer">
+                    <group>
+                        <field name="manufacturer" context="{'default_supplier':True, 'default_customer':False}"/>
```
Comment (lfreeke) [inline product_manufacturer/views/product_manufacturer_view.xml]: It seems this context is not working, I can select customers and suppliers now.

Comment (guewen) [reply product_manufacturer/views/product_manufacturer_view.xml]: Right, odoo behaves differently in this regard now: it always shows everything (suppliers and customers), but orders them by putting suppliers (or customers) first. The new context key to use is `{'res_partner_search_mode': 'supplier'}`. I fixed this.

Thanks for the review!

Comment (lfreeke) [reply product_manufacturer/views/product_manufacturer_view.xml]: Thanks! Works now.

**Review by lfreeke — APPROVED** [review]
Functional test :+1:

**Review by AdriaGForgeFlow — APPROVED** [review]
Code and functional review :+1:

**Pull request merged**